docs/setup-cluster/deploy-cluster/on-prem/linux-packages.rst

```diff
@@ -77,20 +88,22 @@ Install PostgreSQL using ``apt`` or ``yum``
       sudo systemctl enable postgresql.service
 
 #. The authentication methods enabled by default may vary depending on the provider of your
-   PostgreSQL distribution. Ensure that an appropriate authentication method is configured in
-   ``pg_hba.conf`` to enable the ``determined-master`` to connect to the database.
+   PostgreSQL distribution. To enable the ``determined-master`` to connect to the database, ensure
+   that an appropriate authentication method is configured in the ``pg_hba.conf`` file.
 
-   When configuring the database connection (:ref:`configure_the_cluster`):
+   When configuring the database connection in :ref:`configure_the_cluster`, note the following:
 
-   -  If you specify the ``db.hostname`` property, a PostgreSQL ``host`` (TCP/IP) connection will be
-      required.
-   -  If you omit the ``db.hostname`` property, a PostgreSQL ``local`` (Unix-domain socket)
-      connection will be required.
+   -  If you specify the ``db.hostname`` property, you must use a PostgreSQL ``host`` (TCP/IP)
+      connection.
+   -  If you omit the ``db.hostname`` property, you must use a PostgreSQL ``local`` (Unix-domain
```

Comment:
```suggestion
   -  If you omit the ``db.hostname`` property, you must use a PostgreSQL ``local`` (Unix domain
```

Reply:
resolved in 
commit 879c5e97b1a92e6ea169fc53aefc69191513c712